client/src/components/Layout/Header/index.jsx

```diff
@@ -2,17 +2,31 @@ import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Menu, Dropdown } from 'antd';
 import { NavLink } from 'react-router-dom';
+import {
+  NotificationContainer,
+} from 'react-notifications';
 import './style.css';
+import axios from 'axios';
+import createNotification from '../../Users/notification';
 
-export default function Header({ userInfo: { user_name, role } }) {
+export default function Header(props) {
+  const { userInfo: { user_name, role }, clearUserInfo } = props;
+  function handleClick() {
```

Comment:
@IsraaSulaiman Since this functional component now has a function created inside it (handleClick), it might make sense to turn it into a class component (i.e. `export default class Header extends React.Component`).  Then handleClick can be a class method, and inside render you can use it: `... <span onClick={this.handleClick}>Logout</span>`

This is more efficient because then React won't be recreating the handleClick() function every time this Header renders / changes.  Instead, it will create one instance of this component in memory, and just one handleClick in memory, for the entire lifetime of this component, no matter how many time it re-renders itself.